tardis/tests/tests_slow/conftest.py

```diff
@@ -57,25 +50,31 @@ def plot_object(request):
     return PlotUploader(request)
 
 
-@pytest.fixture(scope="session")
-def integration_tests_config(request):
-    return request.config.option.integration_tests_config
+def setup_names():
```

Comment:
It throws an error, that a function is not iterable. I am still confused about a solution for it.